### ISSUE TITLE
modifier+click does not work

### ISSUE THREAD

**Issue by OSS542** [rvaiya/warpd#193]
"Modifier+click" does not work (ex. in thunderbird mail client)
modifier = control or shift

**Comment by rvaiya** [comment]
Control clicking works for me on the tip of master. `M` is bound to `middle` by default.

**Comment by OSS542** [comment]
is not "M' bound to left ? (left click)

**Comment by OSS542** [comment]
I am trying to use control + left click.  I have just tested again on thunderbird, and it does not appear to work.  Deactivating warpd and using the mouse does work.  I'll do more testing and see what I can find out.

**Comment by OSS542** [comment]
Shift + left click (M) does not appear to work either for warpd in normal mode.  It does cause the red block cursor to move downward a significant amount.

**Comment by rvaiya** [comment]
I mean `M` (`S-m`) is mapped to `middle`. `m` is mapped to left mouse down. Control clicking should work as of 04d9f78. What commit are you building from?

**Comment by OSS542** [comment]
I had not realized that 'M' (S-m) was also mapped by warpd.  The thunderbird email client uses  shift+left-click for selecting a range of emails, so that's a conflict with thunderbird rather than a warpd bug.   I've retested control+m with warpd active and noticed a timeout with regard to holding the control key too long before pressing the 'm' key.  It does work correctly if I do not hold the control key too long.  Closing this.
